### DISCUSSION TITLE
2.8.1

### DISCUSSION THREAD

**Discussion by leeyuentuen** [leeyuentuen/alfen_wallbox#118] [Announcements]
## What's Changed
* a proper fix for #107 by @ikruglov in https://github.com/leeyuentuen/alfen_wallbox/pull/112
* prevent infinite loop in _get_all_properties_value() by @ikruglov in https://github.com/leeyuentuen/alfen_wallbox/pull/113
* scan interval config by @leeyuentuen in https://github.com/leeyuentuen/alfen_wallbox/pull/116
* Add RFID-tagged person (who is authorized): https://github.com/leeyuentuen/alfen_wallbox/issues/94
* Add transaction information like last charge/charged time/kWh: https://github.com/leeyuentuen/alfen_wallbox/issues/94
* Load balancing max value increased: https://github.com/leeyuentuen/alfen_wallbox/issues/101

**Full Changelog**: https://github.com/leeyuentuen/alfen_wallbox/compare/2.7.29...2.8.1

<hr /><em>This discussion was created from the release <a href='https://github.com/leeyuentuen/alfen_wallbox/releases/tag/2.8.1'>2.8.1</a>.</em>

**Comment by myhacsint** [comment]
@leeyuentuen Very strange - with this version I can't login anymore - I deleted the Integration entry and tried to recreate, now every time i get:
Unexpected error on GET 401, message='Unauthorized ', url=URL('https://192.168.0.19/api/prop?cat=generic&offset=0')
Unexpected error on GET 401, message='Unauthorized ', url=URL('https://192.168.0.19/api/transactions?offset=0')
Unexpected error on GET unexpected character: line 1 column 14 (char 13)
I can log in with the myEve app with the same password without problems.

**Reply by myhacsint** [reply]
@leeyuentuen Ok the error now changed - apparently the above was a persistent connection that needed some time to timeout - the error now changed to:

Error setting up entry 192.168.0.19 for alfen_wallbox
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 551, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/alfen_wallbox/__init__.py", line 54, in async_setup_entry
    await device.async_update()
  File "/config/custom_components/alfen_wallbox/alfen.py", line 175, in async_update
    await self._get_transaction()
  File "/config/custom_components/alfen_wallbox/alfen.py", line 396, in _get_transaction
    if self.latest_tag[socket,"start","kWh"] is not None:
       ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
KeyError: ('socket 1', 'start', 'kWh')

**Reply by leeyuentuen** [reply]
crap. it is something i changed today. i'll fix soon

**Reply by leeyuentuen** [reply]
@myhacsint  check this public release: https://github.com/leeyuentuen/alfen_wallbox/releases/tag/2.8.2

**Reply by myhacsint** [reply]
@leeyuentuen That seems to work - thanks for the quick fix!!!
